src/app/roles.tsx

```diff
@@ -0,0 +1,28 @@
+"use client";
+
+import React, { useState, createContext } from "react";
+
+interface Props {
+    children: React.ReactNode;
+}
+
+interface RolesInterface {
+    [key: string]: string[];
+}
+
+export const rolesAndHiddenPages: RolesInterface = {
+    admin: [],
+    caller: ["/admin", "/lists"],
+};
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
will there still be an error if you use underscore at the front? (i.e. `_role`)

Reply:
Will check!

Reply:
there shouldn't be... this is the `role` in the (function input) rather than the outside one!